src/App.tsx

```diff
@@ -23,9 +24,8 @@ function ServerMessage() {
 function App(): ReactElement {
   return (
     <div className="App">
-      <MenuItem />
-      <MenuItem />
-      <MenuItem />
+      <AddMealForm />
+      {/* <MenuItem /> */}
```

Comment:
Is there any reason to not deleting it but commenting?

Reply:
Sorry, that's leftovers from the previous branch.